ob-pattern-2/templates/obam/wso2ob-pattern-2-obam-deployment.yaml

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wso2ob-pattern-2-obam-deployment
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: {{ default 1 .Values.wso2.deployment.wso2obam.replicas }} 
```

Comment:
@kaumini @kdanW can you please remove the `default` values being set at the resource definitions?

We can have the default values set at one location (i.e. only at the `values.yaml` file).

Reply:
Fixed with 68829a83552a35f361155eb8c7c65c46e7fe2711